src/osdc/Objecter.cc

```diff
@@ -2247,13 +2248,27 @@ void Objecter::tick()
   if (num_homeless_ops || !toping.empty()) {
     _maybe_request_map();
   } else if (last_osdmap_request_time != ceph::coarse_mono_clock::time_point()) {
+    const epoch_t epoch = osdmap->get_epoch() ? osdmap->get_epoch()+1 : 0;
```

Comment:
Spaces around +

Reply:
in case we don't have any osdmap yet, and the init state of osdmap epoch is 0, we don't want to get an incremental osdmap of 1, we want the full osdmap